cypress/integration/login.js

```diff
@@ -20,5 +20,6 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('[data-test="timesheets-current-day"]').contains('15')
```

Comment:
There is actually a bug in the '[data-test="timesheets-current-day"], and it is not working as expected, you can try changing date to 14, 16, it shouls still pass, as it is not what we would expect

Also regarding todays date, your solution will not work tomorrow, please use todays date
https://www.w3schools.com/jsref/jsref_getdate.asp

Reply:
https://www.w3schools.com/jsref/jsref_getdate.asp

Reply:
👍

Reply:
There is unfortunately a bun in our system, where every calendar day element contains the same `'[data-test="timesheets-current-day"]'`, meaning that this test will check all calendar days and if it finds a day that has a number you give it, e.g. 15, it will pass. It would be better to get the 'calendar--today' class, as in `cy.get('.calendar--today')`.